src/VisualStudio/Core/Def/Implementation/Snippets/AbstractSnippetExpansionClient.cs

```diff
@@ -299,12 +344,16 @@ public virtual bool TryHandleTab()
         {
             if (ExpansionSession != null)
             {
-                var tabbedInsideSnippetField = VSConstants.S_OK == ExpansionSession.GoToNextExpansionField(0);
+                var isFullMethodCallSnippet = !_symbols.IsDefault;
+                var tabbedInsideSnippetField = VSConstants.S_OK == ExpansionSession.GoToNextExpansionField(fCommitIfLast: isFullMethodCallSnippet ? 1 : 0);
```

Comment:
not really understanding the `fCommitIfLast` part, can you doc?

Reply:
⏱️ Adding docs